login-fire-form.html

```diff
@@ -366,6 +389,11 @@
         this.password = null;
       } else {
         this.set('_lastError', e.detail.message);
+
+        if(e.detail.code == 'auth/wrong-password'
+          || e.detail.code == 'auth/user-not-found  ') {
+          this._cleanPasswordInput();
+        }
```

Comment:
Usually, when the entered password is wrong (on OS login screen and websites) the input is focused but not cleared.

In the same thought, for the _"auth/user-not-found"_ error the meaning is the user account does not exists. So if we want to take action it could be to empty both inputs, email and password (maybe only the email, hoping the password is right).

What do you think about this point guys?